tools/codegen/dest/lazy_ir.py

```diff
@@ -251,7 +251,7 @@ def this_shape(i: int) -> str:
         meta_str += f"""
         TORCH_INTERNAL_ASSERT(shapes.size() == {returns_length});"""
 
-        node_str = f"""auto node = torch::lazy::MakeNode<ir::ops::{schema.node_name}>({node_ctor_input_str},
+        node_str = f"""auto node = torch::lazy::MakeNode<{schema.node_name}>({node_ctor_input_str},
```

Comment:
Why was this changed to exclude the namespace? Should nodes no longer be under the `ir::ops` namespace?

Reply:
yea, generally we've simplified the namespaces when landing to master.  The ir::ops. namespace was inherited from the torch/xla project.  We use a flatter namespace, almost everything is just under torch::lazy:: now.